CHANGELOG.md

```diff
@@ -9,6 +9,11 @@
   - Append new items to make git merging easier.
 </details>
 
+## v0.72.2
+- refactor(site): use latest react-styleguidist
+- refactor(site): remove banners on site
+- refactor(site): replace custom "toggle code" functionality with built-in "show code" functionality
```

Comment:
I think this is more of a feature than a refactor but even saying that is genuinely splitting hairs